web/resources/js/tab-anchor.ts

```diff
@@ -26,7 +26,7 @@ function updateLocationHash(this: HTMLElement, _: MouseEvent){
     }
 }
 
-document.addEventListener('DOMContentLoaded', () => {
+window.addEventListener('load', () => {
     navigateToAnchorTab();
```

Comment:
This line works at `DOMContentLoaded` already, we could add another event listener if desired.